sys/include/tsrb.h

```diff
@@ -67,6 +67,17 @@ static inline void tsrb_init(tsrb_t *rb, uint8_t *buffer, unsigned bufsize)
     rb->writes = 0;
 }
 
+/**
+ * @brief        Reset a tsrb.
+ * @param[out]   rb        Ringbuffer to operate on
+ */
+static inline void tsrb_reset(tsrb_t *rb)
```

Comment:
Just nitpicking the name, maybe `tsrb_clear` would be better?

Reply:
Thought about it too and personally I would expect that `clear` just clears bytes and `reset` also resets the read and write pointer, which is the case here 🤷

Reply:
But no strong preference on my side

Reply:
+1 for `tsrb_clear()`. The read and write pointers are implementation details that should be never accessed from users.

Reply:
Then lets go with `clear`, you can squash that right away.